src/config.nim

```diff
@@ -30,8 +30,15 @@ template getReportTemplate*(): ReportTemplate =
   let
     outconf  = chalkConfig.outputConfigs[getBaseCommandName()]
     tmplName = outconf.report_template
+    actionId = "_ACTION_ID"
+  let templateRef = chalkConfig.reportTemplates[tmplName]
```

Comment:
@viega not sure if this is the cleanest spot to do this or if I should just use `forceReportKeys` on `src/reporting.nim`

Reply:
wasnt force keys meant for this use-case where some keys are forced to be always reported like metadata id?